userdetails-plugin/grails-app/views/registration/passwordResetCognito.gsp

```diff
@@ -36,8 +36,10 @@
 <div class="row">
     <h1><g:message code="password.reset.description" /></h1>
 
-    <g:render template="passwordPolicy"
+    <g:if test="${grailsApplication.config.getProperty('userdetails.features.requirePasswordForUserUpdate', Boolean, true)}">
```

Comment:
Wouldn't we want the password policy displayed for cognito resets?

Reply:
Fixed.